scripts/build/Platform/Windows/build_config.json

```diff
@@ -36,6 +36,28 @@
       "test_cpu_profile_vs2019"
     ]
   },
+  "periodic_test_profile_vs2019_pipe": {
+    "TAGS": [
+      "nightly-incremental",
+      "nightly-clean"
+    ],
+    "steps": [
+      "aws_integration_deployment",
+      "periodic_test_profile_vs2019"
```

Comment:
this step includes all automation test? is there a way to run aws specific automation test only?

Reply:
What I'm doing is to remove periodic_test_profile_vs2019 from the nightly build but add this new pipeline instead. We want to make sure that the new job is executed before the periodic_test to make sure that AWS resources are deployed ahead. In this way, we don't need to separate AWS tests and put them in a new job/pipeline.